### PR TITLE
Added tlg0357.tlg001.1st1K-grc2.xml

### PR DESCRIPTION
**Pull request by sonofmun**
Sections of Didymus Alexandrinus Mensurae marmorum ac lignorum

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |       +5 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |      +97 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0357/__cts__.xml`                          | New      |
| `./data/tlg0357/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0357/tlg001/tlg0357.tlg001.1st1K-grc2.xml` | New      |
| `./data/tlg0357/__cts__.xml`                          | Passing  |
| `./data/tlg0357/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0357/tlg001/tlg0357.tlg001.1st1K-grc2.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3725)*

File: data/tlg0357/tlg001/tlg0357.tlg001.1st1K-grc2.xml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+         <name>Frederick Hultsch</name>
+        </persName>
+       </editor>
+       <author ref="urn:cts:greekLit:tlg0559" xml:lang="lat">Didymus Alexandrinus</author>
+       <imprint>
+        <publisher>Teubner</publisher>
+        <pubPlace>Leipzig</pubPlace>
+        <date>1864</date>
+       </imprint>
+      </monogr>
```
Comment (planatheisa) [inline data/tlg0357/tlg001/tlg0357.tlg001.1st1K-grc2.xml]: Scan missing.

File: data/tlg0357/tlg001/tlg0357.tlg001.1st1K-grc2.xml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+   </langUsage>
+  </profileDesc>
+  <revisionDesc>
+   <change/>
+  </revisionDesc>
+ </teiHeader>
+<text>
+<body>
+ <div type="edition" xml:lang="grc" n="urn:cts:greekLit:tlg0357.tlg001.1st1K-grc2">
+<div type="textpart" subtype="chapter" n="12">
```
Comment (planatheisa) [inline data/tlg0357/tlg001/tlg0357.tlg001.1st1K-grc2.xml]: Is there a logic to how these are numbered? These don't make any sense to me.

**Comment by sonofmun** [comment]
The numbering is based on the numbering in the full version of this file. This author took certain portions out of the original to add to the excerpts here on meter. Since we have the full work in another edition, I was able to match up the chunks here to what is there. I wish the editor (who also was the editor of the other edition) would have done it.

**Comment by capitains-hook** [comment]
## Global

| Changed         |   Status |
|:----------------|---------:|
| `nodes_count`   |       +5 |
| `texts_passing` |       +1 |
| `texts_total`   |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |      +97 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0357/tlg001/tlg0357.tlg001.1st1K-grc2.xml` | New      |
| `./data/tlg0357/tlg001/tlg0357.tlg001.1st1K-grc2.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3785)*

**Pull request merged**